liquidity/liquidity_test.go

```diff
@@ -1564,21 +1564,6 @@ func TestFeePercentage(t *testing.T) {
 				DisqualifiedPeers: noPeersDisqualified,
 			},
 		},
-		{
-			name:   "prepay too high",
-			feePPM: 30000,
-			quote: &loop.LoopOutQuote{
```

Comment:
I wonder if instead of deleting just fixing the outcome and adding a comment why this will result in a suggestion is better.